app/build.gradle

```diff
@@ -20,23 +20,38 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    flavorDimensions "default"
+    productFlavors {
+        mock {
+            dimension "default"
+            applicationIdSuffix = ".mock"
+        }
+        prod {
+            dimension "default"
+        }
+    }
+    android.variantFilter { variant ->
+        if (variant.buildType.name == 'release'
+                && variant.getFlavors().get(0).name == 'mock') {
+            variant.setIgnore(true)
+        }
+    }
+
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     implementation "com.android.support:appcompat-v7:$support_library_version"
     implementation "com.android.support:recyclerview-v7:$support_library_version"
     implementation "com.android.support:cardview-v7:$support_library_version"
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
please keep moving this version into the ```buildscript``` as you did with ```kotlin_version``` and ```support_library_version```

Reply:
done (commit [#1d1c28f](https://github.com/diogojs/tmdb_app/pull/1/commits/1d1c28fc7287c9b4bce334240b930e744a7e23e1) )